samples/TEE_aware/dotNET/Makefile

```diff
@@ -0,0 +1,36 @@
+ifdef STRACE
+	OPTS = --strace
+endif
+
+.PHONY: build run clean
+
+all: build
+
+build:  ext2rootfs package
+
+appdir:
+	echo "Creating an appdir from Dockerfile"
+	@myst-appbuilder Dockerfile
+
+ext2rootfs: appdir
+	echo "Building a ext2 file system to run in Mystikos"
+	@myst mkext2 appdir ext2rootfs
+
+runexec: ext2rootfs
+	echo "Running application in ext2 file system with myst exec-sgx"
+	@myst exec-sgx $(OPTS) ext2rootfs /app/TEEAware --app-config-path config.json
```

Comment:
I suggest we use `--roothash=roothash` option for ext2. Otherwise we'd have to explain the security compromise by running Mystikos without it.

Reply:
Fixed